react/queries/productSearchV3.gql

```diff
@@ -35,7 +35,7 @@ query productSearchV3(
     fuzzy: $fuzzy
     operator: $operator
     searchState: $searchState
-  ) @context(provider: "vtex.search-graphql") {
+  ) @connection(key: "productSearch", filter: ["to", "from"]) @context(provider: "vtex.search-graphql")  {
```

Comment:
I think the right filters are: `query`, `fullText`, `orderBy`, `selectedFacets`, `hideUnavailableItems`, because if we change one of those fields, it should use a different cache.